tests/unit/plugin.spec.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @copyright Copyright (c) 2023 Ferdinand Thiessen <opensource@fthiessen.de>
+ *
+ * @author Ferdinand Thiessen <opensource@fthiessen.de>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { NextcloudVuePlugin } from '../../src/plugin'
+import { createLocalVue, mount } from '@vue/test-utils'
+
+describe('Nextcloud Vue Plugin', () => {
+	'use strict'
+
```

Comment:
Does `'use strict'` take effect in ESM?

Reply:
No ESM is always in strict mode. But I am not sure if we are running in ESM mode here as the Typescript is transpiled to JS and we currently have an implicit package type of CommonJS.

Reply:
It should add `use strict` on transtiling. If it doesn't, we should enable it :D